src/app/api/actions/ajouterUneNoteDeContexteAction.test.ts

```diff
@@ -0,0 +1,87 @@
+
+import { revalidatePath } from 'next/cache'
+import { describe, it } from 'vitest'
+
+import { ajouterUneNoteDeContexteAction } from './ajouterUneNoteDeContexteAction'
+import { AjouterNoteDeContexteAGouvernance } from '@/use-cases/commands/AjouterNoteDeContexteAGouvernance'
+
+describe('ajouter une note de contexte', () => {
+  it('quand une note de contexte est ajoutée avec tous ses champs, alors cela renvoie un succès et le cache de la page appelante est purgé', async () => {
+    // GIVEN
+    vi.mock('next/cache', () => ({
+      revalidatePath: vi.fn(),
+    }))
+    vi.spyOn(AjouterNoteDeContexteAGouvernance.prototype, 'execute').mockResolvedValueOnce('OK')
+
+    // WHEN
+    const messages = await ajouterUneNoteDeContexteAction({
+      contenu: '<p>ma note de contexte</p>',
+      path: '/gouvernance/11',
+      uidGouvernance: 'uidGouvernance',
+      uidUtilisateurCourant: 'uidUtilisateurCourant',
+    })
+
+    // THEN
+    expect(messages).toContain('OK')
```

Comment:
```suggestion
    expect(messages).toStrictEqual(['OK'])
```
`toContain` n'est pas assez précis contrairement à `toStrictEqual`.
N'hésite pas à regarder les autres tests pour comparaison